Code/PgSQL/rdkit/expected/rdkit-91.out

```diff
@@ -922,7 +922,7 @@ select 'C1C(C)C1CCCC'::mol @> mol_adjust_query_properties('C1CC1CC'::mol,'{"adju
 (1 row)
 
 select 'C1C(C)C1CCCC'::mol @> mol_adjust_query_properties('C1CC1CC'::mol,'{"adjustDegreeFlags":"bogus"}');
-ERROR:  bad which string provided 'BOGUS'
+ERROR:  Unknown flag value: 'BOGUS'. Flags ignored.
```

Comment:
This implies that all flags are ignored which I think is not true.  Only *this* flag is ignored.

Reply:
If any of the `adjustDegreeFlags` are bad, then they are all ignored.

Reply:
All right, I rescind my comment.